rest/model/pod.go

```diff
@@ -125,6 +125,9 @@ func (p *APIPod) ToService() (*pod.Pod, error) {
 	}
 	timing := p.TimeInfo.ToService()
 	resources := p.Resources.ToService()
+	if err != nil {
```

Comment:
Where is this error coming from?

Reply:
Sorry that's from an old piece of code. I removed it.